### PR TITLE
Angular/Drawer template styling

### PR DESCRIPTION
**Pull request by vegeorgi**

### CODE REVIEW

File: Angular/DrawerNavigation/app/browse/browse.component.html [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+        <Label text="&#xf0c9;" class="fa p-r-15"></Label>
+    </ActionItem>
+    <Label class="action-bar-title" text="Browse"></Label>
 </ActionBar>
 
 <RadSideDrawer #drawer showOverNavigation="true" [drawerTransition]="sideDrawerTransition" tkExampleTitle tkToggleNavButton>
     <StackLayout tkDrawerContent>
         <StackLayout>
-            <MyDrawer></MyDrawer>
+                <MyDrawer [selectedPage]="'Browse'"></MyDrawer>
```
Comment (MartoYankov) [inline Angular/DrawerNavigation/app/browse/browse.component.html]: a minor indentation problem? It's on every page.

File: Angular/DrawerNavigation/app/shared/drawer.component.html [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+        <Label class="sidedrawer-header-brand" text="Name of My App"></Label>
+        <Label class="footnote" text="username@mail.com"></Label>
+    </StackLayout>
+    <StackLayout class="sidedrawer-content">
+        <ListView [items]="items" separatorColor="transparent">
+            <ng-template let-item="item" let-i="index" let-odd="odd" let-even="even">
+                <GridLayout rows="*" columns="auto, *"
+                    (tap)="navigateToRoute(item.route)"
+                    class="sidedrawer-list-item" 
+                    [class.selected]="item.title===selectedPage">
```
Comment (MartoYankov) [inline Angular/DrawerNavigation/app/shared/drawer.component.html]: According to Angular guidelines, this shouldn't be here. There should be no presentation logic in the markup.

"Do put presentation logic in the component class, and not in the template."

**Review by MartoYankov — CHANGES_REQUESTED** [review]
bump

File: Angular/DrawerNavigation/app/browse/browse.component.html (hunk 1 above)
Comment (vegeorgi) [reply Angular/DrawerNavigation/app/browse/browse.component.html]: Done :)

File: Angular/DrawerNavigation/app/shared/drawer.component.html (hunk 2 above)
Comment (vegeorgi) [reply Angular/DrawerNavigation/app/shared/drawer.component.html]: Done :)

**Review by ADjenkov — APPROVED** [review]

**Pull request merged**